lessons/taquito/05/05.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: How much does it cost?
+chapter: Chapter 5
+slug: module-04/chapter-05
+filterBy: lesson-4
+editor:
+  language: javascript
+  startingCode: |
+    Tezos.estimate.originate({
+        // Pass in the CODE_JSON as `code` and STORAGE_JSON as `init`.
+        
+        
+      })
+      .then(operation => {
+        // Print to the console `totalCost` required for the origination of the contract.
+        
+      })
+
+  answer: |
+    Tezos.estimate.originate({
+        // Pass in the CODE_JSON as `code` and STORAGE_JSON as `init`.
+        code: CODE_JSON,
```

Comment:
For a solution we can add our own example michelson code that would make it more clear. Currently, it doesn't throw any error nor does show any thing which is confusing.